src/Directory.Build.props

```diff
@@ -12,6 +12,7 @@
 		<RepositoryType>git</RepositoryType>
 		<PackageTags>SQL XPLAT</PackageTags>
 		<DebugType>portable</DebugType>
+		<Authors>Microsoft</Authors>
```

Comment:
We already have this at the root build props. You shouldn't need to re-add it

https://github.com/microsoft/sqltoolsservice/pull/2173/files#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16eL7